src/edu/ucsd/library/xdre/tab/FilesChecker.java

```diff
@@ -154,6 +172,10 @@ public boolean filesValidate ()
 			// Build the command for file validation
 			List<String> cmd = new ArrayList<>();
 			cmd.add( command );
+			if (src.length() > MAX_FILESIZE) {
```

Comment:
Is there any reason not to just enable large file support?  It would be a little simpler to always enable it, and AFAIK, there's no downside.